src/main/java/net/education/kyivstar/config/ConfigDataBase.java

```diff
@@ -0,0 +1,81 @@
+package net.education.kyivstar.config;
+
+import net.education.kyivstar.services.util.Utils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.yaml.snakeyaml.Yaml;
+
+import java.util.Map;
+import java.util.Optional;
+
+import static java.lang.String.valueOf;
+import static net.education.kyivstar.services.util.ConfigSchemaPassUtils.*;
+import static net.education.kyivstar.services.util.Utils.createDirectoryIfNotExists;
+
+public class ConfigDataBase {
+    private String url;
+    private int port;
+    private String dbName;
+    private String directory;
+    private String user;
+    private String password;
+
+    private static final Logger logger = LoggerFactory.getLogger(ConfigDataBase.class);
+
+    public ConfigDataBase() {
+        loadConfig();
+        createDirectoryIfNotExists(valueOf(getDirectory()));
+    }
+
+    public String getUser() {
+        return user;
+    }
+
+    public String getPass() {
+        return password;
+    }
+
+    public String getDirectory() {
+        return directory;
+    }
+
+    public String getUrl() {
+        return url;
+    }
+
+    public int getPort() {
+        return port;
+    }
+
+    public String getDbName() {
+        return dbName;
+    }
+
+    public void loadConfig() {
+        try (var inputStream = Utils.class.getClassLoader().getResourceAsStream(CONFIG_PATH)) {
```

Comment:
Define constants like CONFIG_PATH, DATABASE, URL, etc., as private static final fields within the class. This ensures that they are scoped to the class and can be easily modified if needed.